docs/manual/installation/update-contao.en.md

```diff
@@ -114,3 +122,114 @@ $ vendor/bin/contao-console contao:migrate
+
+
+{{% notice tip %}}
+The procedure is not limited to updating. You can also use it for a new Contao installation.
+{{% /notice %}}
+
+
+#### Requirements when using the command line
+
+What do you need on your computer?
+
+- Any directory you work in (your working directory)
+- PHP, ideally in the same version as used on your hosting
+- Composer (we assume here that you install Composer in your working directory)
+- Copies of the `composer.json` and `composer.lock` of the Contao installation at your hoster
+
+What do you _not_ need?
+
+- MySQL
+- A local Contao installation
+
+
+#### Performing the update
+
+Copy the `composer.json` and `composer.lock` from your hosting to your working directory.
+You then do essentially the same as described above in [Updating from the command line](#update-via-the-command-line):
+ 
+Open a terminal and change to the working directory. Lead there
+
+```bash
+php composer.phar update
+```
+
+out. After the update has been successfully completed, copy the updated `composer.lock`.
+(and the `composer.json` if you made changes there) back to the Contao installation on your hosting. 
+
+After that you either log in via `ssh` on your server (hosting)
+
+```bash
+ssh benutzername@example.com
+```
+
+and lets Composer install the updated packages
+
+```bash
+php composer.phar install
+```
+
+or you use the Contao Manager. There, under "System maintenance", select the item »Composer dependencies«, »Installer 
+execute«.
+
+![Paketinstallation mit dem Contao-Manager](/de/installation/images/de/composer-install-mit-dem-contao-manager.png?classes=shadow)
+
+Finally you have to update the database tables. 
+
+
+#### Update database tables
+
+Open the [Contao-Installtool](/en/installation/contao-installtool/) and check if any changes have been made to your 
+database are necessary. If necessary, make the suggested changes and then close the install tool.
+
+Instead of the Contao install tool, you can (from Contao 4.9 on) use the Command line the command 
+
+```bash
+$ vendor/bin/contao-console contao:migrate
+``` 
+
+Your Contao installation is now up to date.
+
+
+#### Different PHP versions
+
+If the PHP version used locally is different from the one you have hosted, you need to change the `composer.json` file 
+to a different one. Specify which PHP version should be used:
+
+```json 
+    ...
+    "config": {
+        "platform": {
+            "php": "7.4.99"
+        }
+    },
+    "require": {
+        ...
+    }
+    ...
+```
+
+
+#### Local updates with the Contao Manager
+
+You need a local Contao installation. In this installation, you install the Contao Manager and run the update like
+described in the section [Updating with the Contao Manager](#update-via-the-contao-manager). 
+
+Make sure that the Composer Resolver Cloud is not used. You do not need it, because you are on 
+can provide your own server with enough memory and thus relieve the cloud.
```

Comment:
```suggestion
Make sure that the Composer Resolver Cloud is not used. You do not need it, because you are on 
your own server with enough memory and can thus relieve the cloud from additional workload.
```